handlers/handlers.go

```diff
@@ -0,0 +1,73 @@
+package handlers
+
+import (
+	"encoding/json"
+	"memoryLane/database"
+	"memoryLane/server"
+	"net/http"
+
+	"github.com/go-chi/chi"
+	"github.com/go-chi/cors"
+	"github.com/sirupsen/logrus"
+)
+
+type Handler struct {
+	http.Handler
+}
+
+func NewHandler(db *database.DB) *Handler {
+
+	r := chi.NewRouter()
+
+	// Basic CORS
```

Comment:
This block was an obvious copy-paste, which is of course ok, but it still reads like it. I'd ditch the extraneous comments and commented out stuff and just leave todo's for things you need to fill in.